### PR TITLE
{lib}[foss/2020b] GDAL v3.2.1, GEOS v3.9.1, PROJ v7.2.1, libgeotiff 1.6.0 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3534.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/656a04c62e21d82732bfd3737b1b0112 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3113.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/e2a0c6764ae231d478ea0b8525624f05 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node2653.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/6ce2b9b14f77ab9109b9b6a70562cffb for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 10 out of 10 (4 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/cb3b69128611b4f243d8b51873f2641e for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 7 out of 7 (4 easyconfigs in total)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.7
See https://gist.github.com/7e4418d3fd744028148e091c97af6ccb for a full test report.

File: easybuild/easyconfigs/g/GDAL/GDAL-3.2.1-foss-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    ('PCRE2', '10.35'),
+    ('PROJ', '7.2.1'),
+    ('libgeotiff', '1.6.0'),
+    ('SciPy-bundle', '2020.11'),
+    ('HDF5', '1.10.7'),
+]
+
+preconfigopts = "sed -e 's/-llapack/\$LIBLAPACK/g' -i.eb configure && "
+configopts = '--with-expat=$EBROOTEXPAT --with-libz=$EBROOTLIBZ'
+configopts += ' --without-hdf4 --with-hdf5=$EBROOTHDF5 --with-netcdf=$EBROOTNETCDF'
```
Comment (verdurin) [inline easybuild/easyconfigs/g/GDAL/GDAL-3.2.1-foss-2020b.eb:45]: @boegel we recently added `hdf4` support. Any reason not to keep that?

Comment (boegel) [reply easybuild/easyconfigs/g/GDAL/GDAL-3.2.1-foss-2020b.eb:45]: Ah, I overlooked that... Will fix.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 6 out of 6 (5 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/ee675ebd2091f25574a20ac8c4bb4991 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 6 out of 6 (5 easyconfigs in total)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.7
See https://gist.github.com/7c288de19bed3406064d50e468ccdd3b for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**